### DISCUSSION TITLE
Demo Editor Source code

### DISCUSSION THREAD

**Discussion by jaysridhar** [GrapesJS/grapesjs#5840] [Ideas]
Can you share the complete source code of the demo editor, the one at http://grapesjs.com/demo.html. I found the [source for the vue](https://github.com/GrapesJS/grapesjs/blob/dev/docs/.vuepress/components/DemoTraits.vue) version, but I am looking for plain JS or React.

Thanks

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/blob/gh-pages/demo.html

**Comment by jaysridhar** [comment]
Thanks!

On Wed, May 1, 2024 at 8:59 AM Artur Arseniev ***@***.***>
wrote:

> https://github.com/GrapesJS/grapesjs/blob/gh-pages/demo.html
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrapesJS/grapesjs/discussions/5840#discussioncomment-9281585>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGWQ3ULYAJZXLXDWEDRH3LZABOQ3AVCNFSM6AAAAABGYGSX3OVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TEOBRGU4DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards
Jay
